### PR TITLE
Fix general UI issues according to proposal

### LINKED ISSUE #5306: Adapt UI according to proposal
### Expected behavior

- Modal border radius should be 16px.
Screenshot 2023-09-08 at 14 38 14

- Spacing between modal title and subtext should be 16px. There is an alignment inconsistency with the "Signed message" and the "input view".
Screenshot 2023-09-08 at 14 38 24

- Only page titles should be 28px , all other page titles should be 24px, check the account, transactions, network, and blocks domain.
Screenshot 2023-09-08 at 14 38 59

- Account name and address should be centralized with avatar. Currently the wallet name and address are at the top aligned. And when an account does not have a name the wallet address should be centralized with the avatar as well.
Screenshot 2023-09-08 at 14 39 31

Screenshot 2023-09-08 at 15 11 54

- The spacing between the account name and wallet address should be reduced to 4px and they should be centralized with the avatar. 
Screenshot 2023-09-08 at 15 13 48

- The "back to wallet" button should be a tertiary button (just a text in Lisk primary color).
Screenshot 2023-09-08 at 15 07 18

- Please review the account addition flow based on this [flow](https://www.figma.com/file/KcrDpvWEKQhdGwNd4CZ5NY/Desktop-Prototype?type=design&node-id=38%3A39674&mode=design&t=dEvvwPjv6NAwTzYz-1). Check the spacing, text colors...
Screenshot 2023-09-08 at 15 05 50

- Update the staking modal to the specs of this [design](https://www.figma.com/file/KcrDpvWEKQhdGwNd4CZ5NY/Desktop-Prototype?type=design&node-id=38%3A45796&mode=design&t=dEvvwPjv6NAwTzYz-1) (font size,font colors, spacing).
Screenshot 2023-09-08 at 14 41 32

- All empty states description/subtexts below the illustration should be 16px regular.
Screenshot 2023-09-08 at 14 47 45

**Error message**
- All error message should show beneath the input field as a text and not as a banner. Check [here](https://www.figma.com/file/KcrDpvWEKQhdGwNd4CZ5NY/Desktop-Prototype?type=design&node-id=4430%3A325745&mode=design&t=V0TA12qQkjxjXXOC-1)

**Error fallback UI**
When there’s an error loading the app anytime I open it, the error fallback UI should not take the whole desktop app, users should still be able to see their nav bars and top nav. As the error might be from a particular network and not necessarily all the app. So this way, users are able to change the network. [Design](https://www.figma.com/file/KcrDpvWEKQhdGwNd4CZ5NY/Desktop-Prototype?type=design&node-id=3329%3A244188&mode=design&t=BsmPIIdyMnI5wKs8-1)

### PR DESCRIPTION
**Pull request by ikem-legend**
### What was the problem?

This PR resolves #5306

### How was it solved?

Fixed stated UI issues
- [x] Network selector border radius
- [x] Verify message spacing
- [x] Page title font size
- [x] Wallet address alignment
- [x] Transaction broadcast "Back to wallet"  button
- [x] Add Account flow font, color and spacing
- [x] Staking flow font, color and spacing
- [x] Empty state description font

### How was it tested?

- Visually
- Unit and E2E tests

### CODE REVIEW

**Review by oskarleonard — APPROVED** [review]

**Review by oskarleonard — DISMISSED** [review]

File: src/modules/account/components/AddAccountForm/AddAccountForm.js [hunk 1]
```diff
@@ -84,7 +87,7 @@ const AddAccountFormContainer = ({
               type="submit"
               disabled={!passphrase.isValid || isSubmitDisabled}
             >
-              {t('Continue to set password')}
```
Comment (ManuGowda) [inline src/modules/account/components/AddAccountForm/AddAccountForm.js]: Revert back?

File: src/modules/pos/validator/components/EditStake/index.js [hunk 2]
```diff
@@ -240,7 +240,7 @@ const EditStake = ({ history, stakeEdited, network, staking }) => {
               )}
               <div className={styles.durationSelect}>
                 <div>
-                  <span>{t('Estimated reward')}</span>
```
Comment (ManuGowda) [inline src/modules/pos/validator/components/EditStake/index.js]: This is correct, design is outdated.

File: src/modules/account/components/AddAccountForm/AddAccountForm.js (hunk 1 above)
Comment (ikem-legend) [reply src/modules/account/components/AddAccountForm/AddAccountForm.js]: Done

File: src/modules/pos/validator/components/EditStake/index.js (hunk 2 above)
Comment (ikem-legend) [reply src/modules/pos/validator/components/EditStake/index.js]: Fixed

**Review by ManuGowda — APPROVED** [review]

**Review by oskarleonard — APPROVED** [review]

**Pull request merged**